content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,97 @@
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and one ethnicity is superior to all other ethnicities (racism).  
+
+or  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them special. 
+ 
+or 
+  
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’. 
+For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice. 
+ 
+or
+  
+> Even if *it is the case* we are uniquely superior, morality requires we care for those more vulnerable than ourselves; not that we treat them as a resource.
+
+## Detail - Analytic
+
+> A Definition of Superior (they are many and varied) : of higher rank, quality, or importance - [Merriam-Webster](http://web.archive.org/web/20190711165845/https://www.merriam-webster.com/dictionary/superior)
+
+
+What most people forget (*or choose to overlook*) in using the term superior is that; *We alone* determine the ranking systems. *We alone* determine what denotes better quality. *We alone* determine what makes one thing more important than another.
+
+> "For instance, on the planet Earth, man had always assumed that he was more intelligent than dolphins because he had achieved so much—the wheel, New York, wars and so on—whilst all the dolphins had ever done was muck about in the water having a good time. But conversely, the dolphins had always believed that they were far more intelligent than man—for precisely the same reasons."  
+> - Douglas Adams *from* Hitchhiker's Guide to the Galaxy, 1979
+
+### Humans are different to other animals
+
+That’s true. Humans have a capacity for complex problem solving that is not present in other animals. But why should a set of unique abilities or characteristics equate to superiority and give us the right to do what we like to those deemed inferior?
+
+> Until recently it was considered that our capacity for abstract thought made us unique. However, “there is growing literature indicating that animals are capable of causal reasoning and abstraction”  
+> [Decision-Making From the Animal Perspective: Bridging Ecology and Subjective Cognition (2019)](https://www.frontiersin.org/articles/10.3389/fevo.2019.00164/full)  
+
+Difference and uniqueness are not synonymous with superiority. 
+
+### The flawed reasoning of Superiority and Domination
+
+> For any X and Y, if X is superior (or morally superior) to Y, then X is morally justified in subordinating Y.
```

Comment:
Can we simplify this a bit?

> If a group A is considered superior to group B, then A is morally justified in subordinating B

Reply:
No (ish)

Don't want to weaken the logic.

I do now have a line in the paragraph before that helps....

could possibly add an

or in plain english

then a s second blockquoted bit like you have?

Changed quite a bit now (you'll probably need to have another look

Reply:
Shall we just keep the plain English one? It's probably more appropriate for our intended audience. It hurts to remove the precision of the technical statement, but it's meant for general consumption, not an academic audience and rather than making their ability to argue better, it makes it worse because they'll either not understand it, ignore it, or worse be put off reading the remainder of this section (or even other articles) because of it.

Reply:
Nope.  Bit of a stickler for me (See I have the odd ones).  Find it difficult to believe anyone will give because of one block quoted bit... especially as the 'In plain english' in main body follows directly.

Reply:
Okay let's look at it another way, what does this sentence do, for you? Help me understand why it's so important :)